### PR TITLE
Implement loading animation of ScholarX archive page

### LINKED ISSUE #1549: Add loading animation to ScholarX Archive page
**Is your feature request related to a problem? Please describe.**
We have to add a loading animation to the ScholarX archive page.
It's take quit longer time to load the all data.

**Are you working on it?**
Yes

### PR DESCRIPTION
**Pull request by janithlahirukariyawasam**
## Purpose

The purpose of this PR is to fix #1549 

## Goals

The goal of this PR is to implement ScholarX archive loading animation

### Preview Link

https://pr-1550-sef-site.surge.sh/scholarx/archive/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by janithlahirukariyawasam** [comment]
@anjula-sack can you review this?

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Works like a charm @janithlahirukariyawasam but it's too big. Let's make it small and add a text under it saying "Loading....."

File: scholarx/archive/styles.css [hunk 1]
```diff
@@ -162,3 +163,30 @@ hr {
...
   }
 }
+
+.loader {
+  margin: auto;
+  border: 12px solid #EAF0F6;
+  border-radius: 50%;
+  border-top: 12px solid rgb(215, 215, 215);
+  width: 125px;
+  height: 125px;
```
Comment (anjula-sack) [inline scholarx/archive/styles.css]: Still too big @janithlahirukariyawasam 
```suggestion
  width: 75px;
  height: 75px;
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]

**Review by anjula-sack — APPROVED** [review]
Good job! @janithlahirukariyawasam. Next, we can link this to the scholarx page. With a heading and a button to visit this. Just like the past programs section.

**Pull request merged**